src/lib/es2015.proxy.d.ts

```diff
@@ -1,21 +1,116 @@
 interface ProxyHandler<T extends object> {
+    /**
+     * A trap method for a function call.
+     * @param target The original callable object which is being proxied.
+     * @param thisArg The `this` argument for the function call.
+     * @param argArray The list of arguments for the function call.
+     */
     apply?(target: T, thisArg: any, argArray: any[]): any;
+
+    /**
+     * A trap for the `new` operator.
+     * @param target The original object which is being proxied.
+     * @param argArray The list of arguments for the constructor.
+     * @param newTarget The constructor that was originally called.
+     */
     construct?(target: T, argArray: any[], newTarget: Function): object;
+
+    /**
+     * A trap for `Object.defineProperty()`.
+     * @param target The original object which is being proxied.
+     * @param p The name or `Symbol` of the property whose description is to be retrieved.
+     * @param attributes The descriptor for the property being defined or modified.

```

Comment:
same: the name `attributes` plus type `PropertyDescriptor` already says everything the comment does.